public/app/features/search/components/ManageDashboardsNew.tsx

```diff
@@ -46,6 +46,7 @@ export const ManageDashboardsNew = React.memo(({ folder }: Props) => {
             value={state.query ?? ''}
             onChange={(e) => stateManager.onQueryChange(e.currentTarget.value)}
             onKeyDown={onKeyDown}
+            // eslint-disable-next-line jsx-a11y/no-autofocus
```

Comment:
this is the one on the list that seems debateable... but was already autoFocus 🤷